src/psyclone/psyGen.py

```diff
@@ -203,7 +199,7 @@ def args_filter(arg_list, arg_types=None, arg_accesses=None, arg_meshes=None,
     :param arg_types: List of argument types (e.g. "GH_FIELD")
     :type arg_types: list of str
     :param arg_accesses: List of access types that arguments must have
-    :type arg_accesses: list of str
+    :type arg_accesses: List of :py:class:`psyclone.core.access_type`.
```

Comment:
s/access_type/AccessType/ I think?

Reply:
We were both wrong, as far as I can tell the right way should be: psyclone.core.access_type.AccessType.

Is there actually a way of testing this? I remembered that I saw a doxyfile in the past, but that has been reverted.

Reply:
I think you'd have to modify the documentation source files to say that you wanted Sphinx to generate the docs for this class. One day we will have an API reference guide and we would check in there that it was generated OK.